src/geocat/f2py/rcm2rgrid_wrapper.py

```diff
@@ -164,10 +163,10 @@ def rcm2rgrid(lat2d, lon2d, fi, lat1d, lon1d, msg=None, meta=False):
 
         fi = xr.DataArray(
             fi.data,
-            # coords={
-            #     fi.dims[-1]: lon2d,
-            #     fi.dims[-2]: lat2d,
-            # },
+        # coords={
+        #     fi.dims[-1]: lon2d,
+        #     fi.dims[-2]: lat2d,
+        # },
```

Comment:
I disagree with this action by the linter.

Reply:
I agree! It was due to a strange behavior of `yapf`, and I fixed it now. FYI: Below is why there was such a behavior:

When we were handling the Google style for `yapf` from inside the style file (`based_on_style = google` in `.style.yapf` in the root directory), there was that weird behavior you pointed out. When I removed that setting from style file and instead put it into `.pre-commit-config.yaml` as an argument of `yapf`  (i.e. `args: ['--style=.style.yapf', '--in-place', '--recursive', '--style', 'google']`), it fixed the thing.

I am still keeping the `yapf` style file, because even though there are a few style configs in it for now, there are many [YAPF Knobs](https://github.com/google/yapf#knobs) that we would be useful in the future.

Reply:
Thanks that's great.